src/styles/index.scss

```diff
@@ -1,3 +1,18 @@
+$day-size: 100px;
+$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$gap: 1px;
+$padding: 10px;
+
+@import './main';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
```

Comment:
The styles for 'body' are well-set for centering content, but ensure that any additional global styles you may need are included. If no more global styles are necessary, this is fine as is. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)